common/lib/common-definitions/api-report/common-definitions.api.md

```diff
@@ -220,8 +220,6 @@ export interface ITelemetryBaseEvent extends ITelemetryProperties {
 export interface ITelemetryBaseLogger {
     // (undocumented)
     send(event: ITelemetryBaseEvent): void;
-    // @deprecated (undocumented)
-    supportsTags?: true;
```

Comment:
@ksbrar Reminder to update BREAKING.md for container-definitions 0.21

Reply:
Actually @ksbrar I'm doing a little PR to revise the BREAKING/UPCOMING md files around tagging, to clarify things since we pivoted the plan partway through.

Reply:
#6830 is in 👌